### PR TITLE
Write documentation for sugar3.graphics.style

### PR DESCRIPTION
**Pull request by ndefilippis**

### CODE REVIEW

**Comment by samdroid-apps** [comment]
Hum, your indenting is a bit off, and breaks the code.  I'll leave comments, but make sure that you are using 4 spaces for indents!

File: src/sugar3/graphics/style.py [hunk 1]
```diff
@@ -36,6 +38,10 @@
 
 
 def _compute_zoom_factor():
+	'''
+	Calculates zoom factor based on size of screen. 
+	Returns double representing fraction of maximum possible screen size
+	'''
```
Comment (samdroid-apps) [inline src/sugar3/graphics/style.py]: Intended 4 spaces to much.

File: src/sugar3/graphics/style.py [hunk 2]
```diff
@@ -46,39 +52,75 @@ def _compute_zoom_factor():
 
 
 class Font(object):
-
+	'''
+	A font defines the style of how the text should be rendered. 
+	
+	Args:
+		desc (str): a description of the Font object
+	'''
```
Comment (samdroid-apps) [inline src/sugar3/graphics/style.py]: Intended 4 spaces to much.

File: src/sugar3/graphics/style.py [hunk 3]
```diff
@@ -46,39 +52,75 @@ def _compute_zoom_factor():
...
 class Color(object):
-
+	'''
+	A Color object defines a specific color. 
+
+	Args:
+		color (str): String in the form #FFFFFF representing the color
+		
+		alpha (double):  transparency of color
+	'''
```
Comment (samdroid-apps) [inline src/sugar3/graphics/style.py]: Intended 4 spaces to much.

File: src/sugar3/graphics/style.py [hunk 4]
```diff
@@ -100,6 +147,12 @@ def get_svg(self):
 
 
 def zoom(units):
+	'''
+	Returns size of units pixels at current zoom level
+	
+	Args:
+		units (int): size of item at full size
+	'''
```
Comment (samdroid-apps) [inline src/sugar3/graphics/style.py]: Intended 4 spaces to much.

File: src/sugar3/graphics/style.py [hunk 5]
```diff
@@ -46,39 +52,75 @@ def _compute_zoom_factor():
...
+		'''
+		Returns GDK standard color
+		'''
         return Gdk.Color(int(self._r * 65535), int(self._g * 65535),
                          int(self._b * 65535))
 
     def get_html(self):
+		'''
+		Returns string in the standard html Color format (#FFFFFF)
+		'''
```
Comment (samdroid-apps) [inline src/sugar3/graphics/style.py]: Intended 4 spaces to much.

File: src/sugar3/graphics/style.py [hunk 6]
```diff
@@ -46,39 +52,75 @@ def _compute_zoom_factor():
...
+        '''
+        Returns color encoded as an int, in the form rgba
+        '''
         return int(self._a * 255) + (int(self._b * 255) << 8) + \
             (int(self._g * 255) << 16) + (int(self._r * 255) << 24)
 
     def get_gdk_color(self):
+		'''
+		Returns GDK standard color
+		'''
```
Comment (samdroid-apps) [inline src/sugar3/graphics/style.py]: Intended 4 spaces to much.

**Comment by ndefilippis** [comment]
I need help with squashing two commits together. Whenever I do it, I get a bunch of conflict markers

**Pull request closed without merging**